docs/tutorials/qml/config_qnn.md

```diff
@@ -89,16 +89,18 @@ For full details on the `ObservableConfig` class, see the [API documentation][qa
 
 ## Defining the QNN from the Configs
 
-To build the QNN, we can now use the `QNN` class.
+To build the QNN, we can now use the `QNN` class. In addition to the three parts of the QNN, we can also specify the `backend`, `diff_mode`, etc. For full details on the `QNN` class, see the [API documentation][qadence.ml_tools.models.QNN] or the documentation on the config constructor [here][qadence.ml_tools.models.QNN.from_configs].
```

Comment:
What are the three parts you're referring to ?

```suggestion
To build the QNN, we can now use the `QNN` class as a `QuantumModel` subtype. In addition to the three parts of the QNN, we can also specify options such as the `backend`, `diff_mode`, etc. For full details on the `QNN` class, see the [API documentation][qadence.ml_tools.models.QNN] or the documentation on the config constructor [here][qadence.ml_tools.models.QNN.from_configs].
```

Reply:
Thanks. Clarified the language.